src/Features/Cart/CartTotal.js

```diff
@@ -1,19 +1,30 @@
-import React from 'react'
-import { keys } from 'lodash'
-import products, { getProductsMap } from '../../App/Main/Product/products'
+import { keys } from 'lodash';
+import PropTypes from 'prop-types';
+
+import products, { getProductsMap } from '../../App/Main/Product/products';
 
 const CartTotal = ({
-    productsInCart,
-    productMap = getProductsMap(products),
+  productsInCart,
+  productMap = getProductsMap(products),
 }) => (
-    <div>
-        Total: $
-        {
-            keys(productsInCart).reduce((total,productId) => {
-                return total + (productMap[productId].price * productsInCart[productId])
-            },0)
-        }
-    </div>
-)
+  <div>
+    Total: $
+    {keys(productsInCart).reduce(
```

Comment:
1. Use `Object.values` and remove `lodash` import
2. Move this operation to constant above

Reply:
Done, but haven't found the way how to move this operation out of the component.